test/zepto.html

```diff
@@ -824,10 +824,15 @@
       },
 
       testReady: function(t){
-        t.assertEqual('hi!', globalVarSetFromReady)
-        t.assertEqual('hi!', globalVarSetFromReady2)
-        t.assertEqual('hi!', globalVarSetFromReady3)
-        t.assertEqual('hi!', globalVarSetFromReady4)
+        t.pause()
+        setTimeout(function(){
+          t.resume(function(){
+            t.assertEqual('hi!', globalVarSetFromReady)
+            t.assertEqual('hi!', globalVarSetFromReady2)
+            t.assertEqual('hi!', globalVarSetFromReady3)
+            t.assertEqual('hi!', globalVarSetFromReady4)
+          })
+        }, 500)
```

Comment:
Why is a timeout of 500 necessary here? Do tests fail without a timeout? Keep in mind that the test suite is already executed after a delay, since the test runner activates only on `window.onload`.

If setTimeout is absolutely necessary here to wait for `$(document).ready(...)`, that's understandable, but then a timeout of `0` instead of `500` would be sufficient.